assets/js/stripe-payment-request.js

```diff
@@ -360,6 +360,25 @@ jQuery( function( $ ) {
 				paymentDetails = cart.order_data;
 			}
 
+			// Prevent displaying Link in the PRBs if disabled in the plugin settings.
+			if ( ! wc_stripe_payment_request_params?.stripe?.is_link_enabled ) {
+				options.disableWallets = [ 'link' ];
+			}
```

Comment:
This block is a duplicate of [lines 370-373](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/e3cd93916d6a5f3df30723e206f23f68ec96473a/assets/js/stripe-payment-request.js#L370-L373). This one can be removed.

Reply:
Whoops! Good catch 😄 Removed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2714/commits/9acc9a7f2d0b57786acd14bcf0f6df4f9a43fdb6